wsd/ClientSession.cpp

```diff
@@ -2021,6 +2045,24 @@ bool ClientSession::handleKitToClientMessage(const std::shared_ptr<Message>& pay
                     }
 
                     docBroker->invalidateCursor(x, y, w, h);
+
+                    // session used for thumbnailing and target already was set
+                    if (_thumbnailSession)
+                    {
+                        if (getThumbnailTarget().empty())
+                        {
+                            // cursor already in correct place
+                            std::ostringstream renderThumbnailCmd;
+                            renderThumbnailCmd << "getthumbnail x=" << x << " y=" << y;
+                            docBroker->forwardToChild(client_from_this(), renderThumbnailCmd.str());
+                        }
+                        else
+                        {
+                            // this is initial cursor position message
+                            // we set new position wait for second message
+                            setThumbnailTarget("");
```

Comment:
It's not clear what the comment is trying to say, but `std::string()` is what is needed here. (But should the initial state of the target be an empty string anyway? The comment is confusing.)